frontend/javascripts/statusbar.js

```diff
@@ -0,0 +1,177 @@
+      <span key="zoom">
+        <span
+          key="zoom-i"
+          className="keyboard-key-icon-small"
+          style={{
+            borderColor: "rgba(255, 255, 255, 0.67)",
+          }}
+        >
+          I
+        </span>{" "}
+        /{" "}
+        <span
+          key="zoom-o"
+          className="keyboard-key-icon-small"
+          style={{
+            borderColor: "rgba(255, 255, 255, 0.67)",
+          }}
+        >
+          O
+        </span>{" "}
+        Zoom in/out
+      </span>
+    );
+  }
+
+  getShortcuts() {
+    const { activeViewport } = this.props;
+    return (
+      <Space size={spaceBetweenItems} style={{ textAlign: "right" }}>
+        <span>
+          <img
+            className="keyboard-mouse-icon"
+            src="/assets/images/icon-mouse-left-drag.svg"
+            alt="Mouse Left Drag"
+            style={{ height: 12 }}
+          />
+          Move
+        </span>
+        <span>
+          <img
+            className="keyboard-mouse-icon"
+            src="/assets/images/icon-mousewheel.svg"
+            alt="Mouse Wheel"
+            style={{ height: 12 }}
+          />
+          Move along 3rd axis
+        </span>
+        {this.getZoomShortcut()}
+        {activeViewport === OrthoViews.TDView && (
+          <span>
+            <img
+              className="keyboard-mouse-icon"
+              src="/assets/images/icon-mouse-right.svg"
+              alt="Mouse Right"
+              style={{ height: 12 }}
+            />
+            Rotate 3D View
+          </span>
+        )}
+      </Space>
+    );
+  }
+
+  getCellInfo(globalMousePosition: ?Vector3) {
+    const segmentationLayerName = Model.getSegmentationLayer().name;
+    const cube = this.getSegmentationCube();
+    const renderedZoomStepForMousePosition = api.data.getRenderedZoomStepAtPosition(
+      segmentationLayerName,
+      globalMousePosition,
+    );
+    const getIdForPos = (pos, usableZoomStep) =>
+      pos && cube.getDataValue(pos, null, usableZoomStep);
+
+    return (
+      <span>
+        Cell:{" "}
+        {globalMousePosition
+          ? getIdForPos(globalMousePosition, renderedZoomStepForMousePosition)
+          : "-"}
+      </span>
+    );
+  }
+
+  getInfos() {
+    const { activeViewport, mousePosition, activeResolution } = this.props;
+    let globalMousePosition;
+    if (mousePosition && activeViewport !== OrthoViews.TDView) {
+      const [x, y] = mousePosition;
+      globalMousePosition = calculateGlobalPos({ x, y });
+    }
+
+    return (
+      <Space size={spaceBetweenItems}>
+        <span>
+          <img
+            src="/assets/images/icon-downsampling.svg"
+            style={{ width: 15, height: 15 }}
+            alt="Resolution"
+          />
+          {activeResolution.join("-")}{" "}
+        </span>
+        <span>Pos: [{globalMousePosition ? this.getPosString(globalMousePosition) : "-,-,-"}]</span>
+        {hasSegmentation() && this.getCellInfo(globalMousePosition)}
```

Comment:
I'd just change this to:
```suggestion
        {this.getCellInfo(globalMousePosition)}
```

and then `return null` in that function if `Model.getSegmentationLayer()` returns null.